README.md

```diff
@@ -48,15 +48,23 @@ Huber, Anders Mörtberg.
 
 This makes it possible to directly represent higher inductive types.
 
-Maintainers
------------
 
-* [Anders Mörtberg](https://staff.math.su.se/anders.mortberg/)
+Reviewing of pull requests
+--------------------------
+If you switch your draft pull request (PR) to 'ready to merge',
+or directly create an open PR,
+we should request a review, by one of the reviewers below.
+If that doesn't happen, you can also request a review yourself,
+to make us aware of the open PR.
 
-* [Andrea Vezzosi](http://saizan.github.io/)
+* [Anders Mörtberg](https://staff.math.su.se/anders.mortberg/)
 
 * [Evan Cavallo](https://staff.math.su.se/evan.cavallo/)
 
-[![Build Status](https://travis-ci.org/agda/cubical.svg?branch=master)](https://travis-ci.org/agda/cubical)
+* [Andrea Vezzosi](http://saizan.github.io/) (*inactive*)
```

Comment:
Maybe move Andrea to the end of the list if he's inactive?

Reply:
Yes, good point.
Also, I didn't know that @mzeuner and  @aljungstrom do reviews - they should be on the list then.

Reply:
I think they both did some after I requested them to do it :-)

Reply:
I added them. Any hints for their topics?

Reply:
algebra for Max and synthetic homotopy/cohomology theory for Axel

Reply:
Added